trollmoves/dispatcher.py

```diff
@@ -318,7 +321,19 @@ def run(self):
                     continue
                 destinations = self.get_destinations(msg)
                 if destinations:
-                    success = dispatch(msg.data['uri'], destinations)
+                    # Check if the url are on another host:
+                    url = urlparse(msg.data['uri'])
+                    if url.scheme != '':
+                        netloc = url.netloc
+                        if netloc != self.host:
+                            # This warning may appear even if the file path
+                            # does exist on both servers (a central file server
+                            # reachable from both). But in those cases one
+                            # should probably not use an url scheme but just a
+                            # file path:
+                            logger.warning(
```

Comment:
Should this raise a `NotImplementedError` ?

Reply:
Yes, good idea!
Done